rchttp/client.go

```diff
@@ -10,17 +11,21 @@ import (
 	"strings"
 	"time"
 
+	"github.com/ONSdigital/go-ns/common"
 	"golang.org/x/net/context"
 	"golang.org/x/net/context/ctxhttp"
 )
 
 // Client is an extension of the net/http client with ability to add
 // timeouts, exponential backoff and context-based cancellation
 type Client struct {
-	MaxRetries         int
-	ExponentialBackoff bool
-	RetryTime          time.Duration
-	HTTPClient         *http.Client
+	MaxRetries           int
+	ExponentialBackoff   bool
+	RetryTime            time.Duration
+	HTTPClient           *http.Client
+	AuthToken            string
+	DownloadServiceToken string
+	FlorenceToken        string
```

Comment:
I can't see why the Florence Token /  DownloadServiceToken are stored within the client here. They don't appear to be used within rchttp.client. It appears that  the florence token is injected into the client as required for requests, so I think we can remove it from the struct / remove the setters?

Reply:
As per our discussion, I will remove FlorenceToken (it's set by `clients/identity` not rchttp)